Docs/source/usage/workflows/ml_dataset_training.rst

```diff
@@ -221,22 +228,25 @@ The testing error serves as a metric of model generalizability, indicating how w
 on data it hasn't seen yet.
 When the test-loss starts to trend flat or even upward, the neural network is no longer improving its ability to generalize to new data.
 
-.. figure:: https://user-images.githubusercontent.com/10621396/290010428-f83725ab-a08f-494c-b075-314b0d26cb9a.png
-   :alt: Plot of training and testing loss curves
+.. _fig_train_test_loss:
 
-   Plot of training (in blue) and testing (in green) loss curves versus number of training epochs.
+.. figure:: fig_train_test_loss.png
+   :alt: Plot of training and testing loss curves versus number of training epochs.
 
-A visual inspection of the model prediction can be seen in Fig. `[fig:train_evaluation]` .
-This plot compares the model prediction, with dots colored by mean-square error, on the testing data with the actual simulation output in black.
+   Training (in blue) and testing (in green) loss curves versus number of training epochs.
+
+.. _fig_train_evaluation:
 
-.. figure:: https://user-images.githubusercontent.com/10621396/290010486-4a3541e7-e0be-4cf1-b33b-57d5e5985196.png
+.. figure:: fig_train_evaluation.png
```

Comment:
Please do not add the PNGs to git, since we want to keep the repo small.

URL-based rendering works well: https://warpx.readthedocs.io/en/latest/usage/workflows/ml_dataset_training.html

Did you see something breaking?

Reply:
Ah okay. Why do we have any PNG files then? Why not make all of the figures links?

Reply:
> Ah okay. Why do we have any PNG files then? Why not make all of the figures links?

This is the plan. People started adding PNGs before I could stop them (aka: before I was hired ;-) )